astropy/coordinates/tests/test_iau_fullstack.py

```diff
@@ -126,51 +99,39 @@ def test_iau_fullstack(
     # now make sure the full stack round-tripping works
     icrs2 = aacoo.transform_to(ICRS())
 
-    adras = np.abs(fullstack_icrs.ra - icrs2.ra)[msk]
-    addecs = np.abs(fullstack_icrs.dec - icrs2.dec)[msk]
-    assert np.all(
-        adras < tol
-    ), f"largest RA change is {np.max(adras.arcsec * 1000)} mas, > {tol}"
-    assert np.all(
-        addecs < tol
-    ), f"largest Dec change is {np.max(addecs.arcsec * 1000)} mas, > {tol}"
+    assert_quantity_allclose(
+        np.abs(fullstack_icrs.ra - icrs2.ra)[msk], 0 * u.μas, atol=tol, rtol=0
+    )
+    assert_quantity_allclose(
+        np.abs(fullstack_icrs.dec - icrs2.dec)[msk], 0 * u.μas, atol=tol, rtol=0
+    )
 
     # check that we're consistent with the ERFA alt/az result
-    iers_tab = iers.earth_orientation_table.get()
-    xp, yp = u.Quantity(iers_tab.pm_xy(fullstack_times)).to_value(u.radian)
-    lon = fullstack_locations.geodetic[0].to_value(u.radian)
-    lat = fullstack_locations.geodetic[1].to_value(u.radian)
-    height = fullstack_locations.geodetic[2].to_value(u.m)
-    jd1, jd2 = get_jd12(fullstack_times, "utc")
-    pressure = fullstack_obsconditions[0].to_value(u.hPa)
-    temperature = fullstack_obsconditions[1].to_value(u.deg_C)
-    # Relative humidity can be a quantity or a number.
-    relative_humidity = u.Quantity(fullstack_obsconditions[2], u.one).value
-    obswl = fullstack_obsconditions[3].to_value(u.micron)
-    astrom, eo = erfa.apco13(
-        jd1,
-        jd2,
+    astrom, _ = erfa.apco13(
+        *get_jd12(fullstack_times, "utc"),
         fullstack_times.delta_ut1_utc,
-        lon,
-        lat,
-        height,
-        xp,
-        yp,
-        pressure,
-        temperature,
-        relative_humidity,
-        obswl,
+        (geodetic := fullstack_locations.geodetic).lon.rad,
```

Comment:
We can leave it to a later PR, but these days the erfa routines can handle units, so no need for the `.rad` and `.to_value`.

Reply:
I have `pyerfa` 2.0.1.4 installed. If I change
```suggestion
        (geodetic := fullstack_locations.geodetic).lon,
```
then the tests fail with
`TypeError: unknown ufunc apco13.  If you believe this ufunc should be supported, please raise an i...`

Reply:
Hmm, that's definitely a bug, so how about leaving this as is for now?

Reply:
Have to teach, so no time to fix this now, but I raised #17738.